x-pack/plugins/observability_solution/infra/public/components/shared/templates/no_data_config.ts

```diff
@@ -107,12 +107,14 @@ export const getNoDataConfig = ({
   docsLink,
 }: {
   hasData: boolean;
+  hasEemData?: boolean;
```

Comment:
We can now remove `hasEemData` and `isEntityCentricExperienceEnabled` from here.

Reply:
Forgot to remove the ones in there, thank you.

https://github.com/elastic/kibana/pull/196387/commits/199f836519a6d5354edcd5f4e717d77e9c63be6a